modules/googleplay/src/main/scala/cards/nine/googleplay/processes/CardsProcesses.scala

```diff
@@ -207,23 +208,32 @@ class CardsProcesses[F[_]](
 
     webScrapper.getDetails(pack) flatMap {
       case Right(card) ⇒
-        cacheService.putResolved(card).map(x ⇒ Resolved)
+        toFree(cacheService.putResolved(card).map(x ⇒ Resolved))
       case Left(PageParseFailed(_)) ⇒
-        cacheService.setToPending(pack).map(x ⇒ Pending)
+        toFree(cacheService.setToPending(pack).map(x ⇒ Pending))
       case Left(PackageNotFound(_)) ⇒
-        cacheService.addError(pack).map(x ⇒ Unknown)
+        toFree(cacheService.addError(pack).map(x ⇒ Unknown))
       case Left(WebPageServerError) ⇒
-        cacheService.setToPending(pack).map(x ⇒ Pending)
+        toFree(cacheService.setToPending(pack).map(x ⇒ Pending))
     }
   }
+
+  def toFree[A](fs: FreeS.Par[F, A]): Free[F, A] = {
```

Comment:
You can use `FreeApplicative#monad` which turns a `FreeApplicative[F, A]` into a `Free[F, A]`